### PR TITLE
adjust subvolume list to have NoCOW /var instead of many subvolumes

### PR DESCRIPTION
**Pull request by sysrich**

### CODE REVIEW

**Comment by sysrich** [comment]
Ping - PR updated with requested comment, please re-review

File: package/skelcd-control-CAASP.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Dec 12 17:11:12 UTC 2017 - rbrown@suse.com
+
+- adjust subvolume list to have NoCOW /var instead of many subvolumes
```
Comment (jreidinger) [inline package/skelcd-control-CAASP.changes]: well, we need some bsc or FATE as it is requested by an IBS bot. And does this change is also planned for CaaSP4? as it is currently shared.

Comment (sysrich) [reply package/skelcd-control-CAASP.changes]: Q1. Which IBS bot? I have made over 207 changes to CaaSP & SLE as part of this endeavour, all of which are now in SLE and Tumbleweed. This is the first I've heard of such a bot.

Q2. skelcd-control-CAASP is also used by Tumbleweed/Kubic (the upstream for SLE/CaaSP) - as the engineer responsible for Kubic I can assure you that no requirements are needed there, but I do need this change.

Comment (jreidinger) [reply package/skelcd-control-CAASP.changes]: @sysrich I do not remember name, but some bot that auto decline if fix reach opensuse earlier then SLE that there is no known issue and it cannot compare if same change is also in other product.
If it is no longer needed I would be more then happy to disable this check.

Comment (sysrich) [reply package/skelcd-control-CAASP.changes]: @jreidinger that doesn't make any sense - openSUSE has hundreds of thousands of changes that do not have bug reports or fates..we don't expect our upstream community to do that.

I know of a bot that works in the opposite direction, if stuff hits SLE before openSUSE without a comparable reference, and there it makes sense - we need to track that to ensure we have SLE-relevant bugs and features present in openSUSE

Comment (jreidinger) [reply package/skelcd-control-CAASP.changes]: ok, you are right. So we implement this checker that ensure that each change as it goes also to SLE have such reference.

**Review by jreidinger — APPROVED** [review]

**Pull request merged**